src/github_actions_exporter.py

```diff
@@ -56,7 +57,7 @@ def is_configuration_valid(state: CharmState) -> bool:
     Returns:
         True if they are all set
```

Comment:
nit: I think this is no longer valid, right?

Reply:
Strictly speaking, the proposition is still true, but the function will also return true if either the webhook token or the other parameters are set.

Reply:
Ops, I forgot to update the comment. I can do it in a further PR.